app/assets/javascripts/components/form/student_form.es6.jsx

```diff
@@ -631,15 +634,12 @@ class StudentForm extends React.Component {
                 {this.displayErrorMessage("gender")}
               </FormGroup>
 
-              <FormGroup validationState={this.getValidationState("birthday")}>
-                <ControlLabel>Birthday</ControlLabel>
-                <Datetime
-                  dateFormat="MM/DD/YYYY"
-                  timeFormat={false}
-                  inputProps={{placeholder: "MM/DD/YYYY"}}
-                  onChange={(moment) => this.handleDatetimeChange(moment, 'birthday')}/>
-                {this.displayErrorMessage("birthday")}
-              </FormGroup>
+              <FormatInput 
+                formName             = "Birthday"
+                inputId              = "birthday"
+                handleDatetimeChange = { (moment) => this.handleDatetimeChange(moment, "birthday") }
```

Comment:
Ah, I know what happened. Originally I had two separate files—PhoneInput and DateInput—but I realized the logic was so similar that it would be better to put them in one file, and so handleDatetimeChange was originally a prop type. I think I'll change the name of the birthday prop type to handleChange, but I'll still pass down the handleDatetimeChange function.